files/es/web/http/headers/index.md

```diff
@@ -0,0 +1,347 @@
+
+La siguiente lista agrupa las cabeceras HTTP en categorías según su uso. Para visualizar una lista en orden alfabético, use el navegador del lado izquierdo.
+
+## Autenticación
+
+- {{HTTPHeader("WWW-Authenticate")}}
+  - : Define el método de autenticación que debería ser usado para tener acceso al contenido.
+- {{HTTPHeader("Authorization")}}
+  - : Contiene las credenciales para autenticar a un usuario con un servidor.
+- {{HTTPHeader("Proxy-Authenticate")}}
+  - : Define el método de autenticación que debería ser usado para tener acceso a un recurso que se encuentre tras un servidor proxy.
+- {{HTTPHeader("Proxy-Authorization")}}
+  - : Contiene las credenciales para autenticar a un usuario con un servidor proxy.
+
+## Almacenamiento en caché
+
+- {{HTTPHeader("Age")}}
+  - : El tiempo en el que un objeto ha estado en una caché proxy, expresado en segundos.
+- {{HTTPHeader("Cache-Control")}}
+  - : Especifica directivas para los mecanismos de almacenamiento en caché, tanto para peticiones como para respuestas.
+- {{HTTPHeader("Expires")}}
+  - : La fecha y tiempo tras las cuales una respuesta se considera agotada.
+- {{HTTPHeader("Pragma")}}
+  - : Cabecera specífica para implementaciones que puede tener diferentes efectos a lo lartgo del proceso de petición-respuesta. Utilizada para implementar retrocompatibilidad con cachés de tipo HTTP/1.0 donde la cabecera `Cache-Control` aún no esté presente.
+- {{HTTPHeader("Warning")}}
+  - : Un campo de alerta general, que contiene i nformación sobre diferentes problemas posibles.
+
+## Indicaciones sobre el cliente
+
+- {{HTTPHeader("Accept-CH")}}
+  - : ...
+
+<!---->
+
+- {{HTTPHeader("Accept-CH-Lifetime")}}
+  - : ...
+- {{HTTPHeader("Early-Data")}}
+  - : ...
+
+<!---->
+
+- {{HTTPHeader("Content-DPR")}}
+  - : ...
+- {{HTTPHeader("DPR")}}
+  - : ...
+- {{HTTPHeader("Downlink")}}
+  - : ...
+- {{HTTPHeader("Save-Data")}}
+  - : ...
+- {{HTTPHeader("Viewport-Width")}}
+  - : ...
+- {{HTTPHeader("Width")}}
+  - : ...
+
+<!---->
+
+- ## Condicionales
+
+  {{HTTPHeader("Last-Modified")}}
+
+  - : Se trata de un validador, indicando la fecha de la última modificación del recurso, utilizado para comparar diferentes versiones del mismo recurso. No es tan preciso como {{HTTPHeader("ETag")}}, pero es más sencillo de calcular en algunos entornos. Las peticiones condicionales que usan {{HTTPHeader("If-Modified-Since")}} y {{HTTPHeader("If-Unmodified-Since")}}utilizan este valor para cambiar el comportamiento de la petición.
+
```

Comment:
```suggestion
## Condicionales

- {{HTTPHeader("Last-Modified")}}
  - : Se trata de un validador, indicando la fecha de la última modificación del recurso, utilizado para comparar diferentes versiones del mismo recurso. No es tan preciso como {{HTTPHeader("ETag")}}, pero es más sencillo de calcular en algunos entornos. Las peticiones condicionales que usan {{HTTPHeader("If-Modified-Since")}} y {{HTTPHeader("If-Unmodified-Since")}}utilizan este valor para cambiar el comportamiento de la petición.
```